Traceback (most recent call last):
  File "file-test-data/extras/testWrongKeywordArg5.py", line 4, in <module>
    foo()

WyppTypeError: Anzahl der Argument passt nicht

Funktion `foo` ben√∂tigt 1 Argument.
Gegeben: keine Argumente

## Datei file-test-data/extras/testWrongKeywordArg5.py
## Aufruf in Zeile 4:

foo()